static/css/tokens.css

```
@@ -6,6 +6,8 @@
 :root {
     /* Perspective: Professional yet warm */
     --font-family: 'Inter', system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+    --font-display: 'Atkinson Hyperlegible Next', 'Inter', system-ui, -apple-system, "Segoe UI", sans-serif;
+    --font-data: 'IBM Plex Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, monospace;
```

Stylelint (17.14.1)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)